src/fu-tool.c

```diff
@@ -3242,6 +3245,52 @@ fu_util_switch_branch(FuUtilPrivate *priv, gchar **values, GError **error)
 	return fu_util_prompt_complete(priv->completion_flags, TRUE, error);
 }
 
+static gboolean
+fu_util_get_bios_attr(FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	g_autoptr(FuBiosAttrs) attrs = NULL;
+	g_autoptr(GPtrArray) items = NULL;
+	FuContext *ctx;
+	gboolean found = FALSE;
+
+	/* load engine */
+	if (!fu_util_start_engine(priv,
+				  FU_ENGINE_LOAD_FLAG_HWINFO | FU_ENGINE_LOAD_FLAG_COLDPLUG,
+				  priv->progress,
+				  error))
+		return FALSE;
+
+	ctx = fu_engine_get_context(priv->engine);
+	attrs = fu_context_get_bios_attrs(ctx);
+	items = fu_bios_attrs_get_all(attrs);
+	if (priv->as_json)
+		return fu_util_get_bios_attr_as_json(values, items, error);
+
+	for (guint i = 0; i < items->len; i++) {
+		FwupdBiosAttr *attr = g_ptr_array_index(items, i);
+		if (fu_util_bios_attr_matches_args(attr, values)) {
+			g_autofree gchar *tmp = fu_util_bios_attr_to_string(attr, 0);
+			g_print("%s\n", tmp);
+			found = TRUE;
+		}
+	}
+	if (items->len == 0) {
+		g_set_error_literal(error,
+				    FWUPD_ERROR,
+				    FWUPD_ERROR_NOTHING_TO_DO,
+				    _("This system doesn't support firmware settings"));
+		return FALSE;
+	}
+	if (!found) {
+		g_set_error(error,
+			    FWUPD_ERROR,
+			    FWUPD_ERROR_INVALID_ARGS,
+			    "Unable to find attribute %s",
+			    values[0]);
+	}
+	return found;
```

Comment:
I'd prefer return FALSE, then return TRUE here -- then if we add another conditional we don't have to do mental accounting as to whether the GError is valid.